requirements/base.in

```diff
@@ -10,4 +10,4 @@ pylint
 pytest
 sgqlc
 websockets
--e git+https://github.com/gridsingularity/gsy-framework@feature/D3ASIM-3669#egg=gsy_framework
```

Comment:
Ouch

Reply:
Can you replace branch name with env var for `gsy-e-sdk` tp be tested in `staging` & `production` too?

Reply:
@faizan2590 can you elaborate more? do you mean that I should use an environment variable to dynamically select the branch here ?

Reply:
> do you mean that I should use an environment variable to dynamically select the branch here ?

Yes

Reply:
Let me try it out, if it's okay I'll work on it after merging this PR if you agree

Reply:
Sure

Reply:
It seems that this is not straightforward, kindly have a look at the answers [here](https://www.py4u.net/discuss/173406).
However, I indeed believe that this something to be fixed, I'll create a ticket for it and place it in the upcoming sprint